### DISCUSSION TITLE
Refresh at index page goes back to login

### DISCUSSION THREAD

**Discussion by ppbolivar20** [ctron/yew-oauth2#16] [Q&A]
When running yew-oauth2-example, a browser refresh at the index page prompts for a login even if user is already authenticated.  But when in other pages, i.e. component, function, etc. everything works fine.  Am I missing something?

**Comment by ctron** [comment]
That doesn't sound right. So what will happen when you reload the page is that the WASM code starts to run again, missing the tokens initially (as they are not stored). That will redirect to the SSO server, which then (if it still has a valid session) just redirect back to the actual page, including the "code", which the WASM code then trades for a set of tokens.

That should work the same for the index page, as well as any other page.

How are you running this?
